### PR TITLE
Instructions for exporter-only install

### PR DESCRIPTION
**Pull request by michaelmdresser**
Depends on https://github.com/kubecost/cost-model/pull/696

Also requires the changes from https://github.com/kubecost/cost-model/pull/693 to be published in the `cost-model` container.

### CODE REVIEW

File: kubecost-exporter.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+If you would prefer to not use the recommended install option and just deploy the Kubecost open source cost model as a metric exporter, you can follow these steps:
+
+
+1. Apply the combined YAML:
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/kubecost/cost-model/develop/kubernetes/exporter/exporter.yaml --namespace cost-model
+    ```
+
+    > If you want to use a namespace other than `cost-model`, you will have to edit the `ClusterRoleBinding` after applying the YAML to change `subjects[0].namespace`. You can do this with `kubectl edit clusterrolebinding cost-model`.
```
Comment (dwbrown2) [inline kubecost-exporter.md:18]: I think most users would just modify the YAML before applying. I personally think it's ok to drop this in open source docs.

File: kubecost-exporter.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+2. To verify that metrics are available:
```
Comment (dwbrown2) [inline kubecost-exporter.md:20]: To me, this feels like just verifying that Step 1 worked. Where as adding the scrape config is the true Step 2. Are you thinking about this differently?

File: kubecost-exporter.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+Done! Kubecost is now exporting cost metrics. See the following sections for different metrics available and query examples.
+
+## Available Prometheus Metrics 
+
+| Metric       | Description                                                                                            |
+| ------------ | ------------------------------------------------------------------------------------------------------ |
+| node_cpu_hourly_cost | Hourly cost per vCPU on this node  |
+| node_gpu_hourly_cost | Hourly cost per GPU on this node  |
+| node_ram_hourly_cost   | Hourly cost per Gb of memory on this node                       |
```
Comment (dwbrown2) [inline kubecost-exporter.md:47]: We now have `kubecost_load_balancer_cost` to add to this list, yeah?

File: kubecost-exporter.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+```
+avg_over_time(container_memory_allocation_bytes{namespace="kube-system",pod=~"fluentd.*"}[1d])
+  * on (pod,node) group_left
+avg(count_over_time(container_memory_allocation_bytes{namespace="kube-system"}[1d:1m])/60) by (pod,node)
+  * on (node) group_left
+avg(avg_over_time(node_ram_hourly_cost[1d] )) by (node)
+```
+
+#### Monthly cost of a label
```
Comment (dwbrown2) [inline kubecost-exporter.md:97]: this is still a TODO as well

File: kubecost-exporter.md (hunk 1 above)
Comment (michaelmdresser) [reply kubecost-exporter.md:18]: My first instinct when following one of our other install instructions was to just modify `kubectl apply` with a different value for the `--namespace` flag.  I ran into this issue and it took me a while to fix it because everything was running but the `cost-model` pod was constantly logging failures (while staying up) because it wasn't getting the right permission scopes and I had to track down why.

Basically, I included this instruction because I would have benefited from it. Definitely interested in a better place for it, because right now I think it visually clutters up the "easy 2 step process" idea.

File: kubecost-exporter.md (hunk 2 above)
Comment (michaelmdresser) [reply kubecost-exporter.md:20]: I think adding the scrape config is definitely the true step 2, but I think I'm in favor of keeping it like this because this is a much more rapid-feedback way of ensuring that everything is running as expected and it gives a user the instinct for what to reach for if they need to debug.

File: kubecost-exporter.md (hunk 3 above)
Comment (michaelmdresser) [reply kubecost-exporter.md:47]: I'm not seeing it with the latest (public) `cost-model` container image. Is that a feature coming in this release?

File: kubecost-exporter.md (hunk 4 above)
Comment (michaelmdresser) [reply kubecost-exporter.md:97]: Right, I think I glossed over it. I'll try to craft this query tomorrow.

File: kubecost-exporter.md (hunk 3 above)
Comment (dwbrown2) [reply kubecost-exporter.md:47]: Should be available for the last 5+ releases: https://github.com/kubecost/cost-model/blob/f845eb02c200c4da993827ad2d62bf47b5922f84/pkg/costmodel/metrics.go#L636

Comment (michaelmdresser) [reply kubecost-exporter.md:47]: Ah, it's probably not there for me because I don't have any load balancers so its not being emitted.

Comment (michaelmdresser) [reply kubecost-exporter.md:47]: Are we documenting these metrics somewhere other than PROMETHEUS.md? It also doesn't have `kubecost_load_balancer_cost`, not to mention others like:
* `kubecost_node_is_spot`
* `kubecost_network_zone_egress_cost`
* `kubecost_network_region_egress_cost`
* `kubecost_network_internet_egress_cost`
* `kubecost_cluster_management_cost`
* `kubecost_cluster_info`

File: kubecost-exporter.md (hunk 4 above)
Comment (dwbrown2) [reply kubecost-exporter.md:97]: Happy to help if you need me! I'd start with the simplest version and then we can add complexity in future queries.

File: kubecost-exporter.md (hunk 3 above)
Comment (dwbrown2) [reply kubecost-exporter.md:47]: We have not documented anywhere yet! In this section, it seems like `kubecost_load_balancer_cost` and `kubecost_cluster_management_cost` are the most relevant? 

I haven't added network_egress metrics yet primarily because they are in a different repo.

**Review by dwbrown2 — APPROVED** [review]
LGTM, feel free to merge when ready or we can wait until just before launch event!

File: kubecost-exporter.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+sum(kubecost_load_balancer_cost) by (namespace)
+```
+
+#### Monthly rate of each namespace’s CPU request
+
+```
+sum(container_cpu_allocation * on (node) group_left node_ram_hourly_cost) by (namespace) * 730
+```
+
+#### Historical spend for all `fluentd` pods in the `kube-system` namespace
```
Comment (michaelmdresser) [inline kubecost-exporter.md:87]: This looks like its actually historical _memory_ spend. Is that right @dwbrown2?

Comment (dwbrown2) [reply kubecost-exporter.md:87]: That's correct! Thanks for catching.

For these initial queries, I just wanted to keep it simple... as opposed to showing mem + cpu + loadbalancer, etc etc.

File: kubecost-exporter.md (hunk 3 above)
Comment (michaelmdresser) [reply kubecost-exporter.md:47]: I'll open a follow-up PR to add these metrics both here and to PROMETHEUS.md
